yt/data_objects/index_subobjects/grid_patch.py

```diff
@@ -142,7 +142,8 @@ def _setup_dx(self):
         self.dds.units = self.index.grid_left_edge.units
 
     def __repr__(self):
-        return "AMRGridPatch_%04i" % (self.id)
+        cls_name = self.__class__.__name__
+        return f"{cls_name}_{self.id:04i} ({self.ActiveDimensions})"
```

Comment:
I think this is fine, and don't want to change it, but I want to note that `self.ActiveDimensions` is the not set in the base class constructor.

Reply:
Ah, my mistake. I can make it conditional then